packages/react-instantsearch-hooks-dom/src/widgets/RefinementList.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useRefinementList } from 'react-instantsearch-hooks';
+
+import { cx } from '../ui/lib/cx';
+import { RefinementList as RefinementListUiComponent } from '../ui/RefinementList';
+import { SearchBox as SearchBoxUiComponent } from '../ui/SearchBox';
+import { ShowMoreButton } from '../ui/ShowMoreButton';
+
+import type { RefinementListProps as RefinementListUiComponentProps } from '../ui/RefinementList';
+import type { RefinementListItem } from 'instantsearch.js/es/connectors/refinement-list/connectRefinementList';
+import type { RefinementListWidgetParams } from 'instantsearch.js/es/widgets/refinement-list/refinement-list';
+import type { UseRefinementListProps } from 'react-instantsearch-hooks';
+
+export type RefinementListProps = Omit<
+  RefinementListUiComponentProps,
+  'items' | 'onRefine' | 'searchBox' | 'showMoreButton'
+> &
```

Comment:
I don't see a use for the sole leftover prop `noResults` here. We could replace all this with `React.HTMLAttributes<HTMLDivElement>` instead?

Reply:
Good catch, I forgot to omit it. Addressed in 17f5cdbc.